doc/user/ref_advanced_func.rst

```diff
@@ -438,6 +440,64 @@ These can be implemented as, for example, to force at most 20% of ``energy_cap``
 
 .. seealso:: The above examples are supplied as overrides in the :ref:`built-in national-scale example <examplemodels_nationalscale_settings>`'s ``scenarios.yaml`` (``cold_fusion`` to define that tech, and ``group_share_cold_fusion_prod`` or ``group_share_cold_fusion_cap`` to apply the group share constraints).
 
+.. _model_wide_constraints:
+
+Model-wide constraints
+----------------------
+
+Model-wide constraints are applied to named sets of locations and techs specified within a top-level ``modelwide_constraints`` key (sitting alongside other top-level keys like ``model`` and ``run``).
```

Comment:
We now have 'systemwide' (e.g. `systemwide_capacity_factor`) and 'modelwide'. I would suggest we collapse this to one, although I have no preference as to which is kept

Reply:
Ok - how about calling them "group constraints", implying that they are applied to user-defined groups of model components. @timtroendle: does that make sense to you?

Reply:
Sounds good to me. I think it's more precise than "model-wide constraints".

Reply:
Done.